spec/core/ics-002-client-semantics/README.md

```diff
@@ -660,29 +569,43 @@ to allow governance mechanisms to perform these actions
 ```typescript
 function updateClient(
   id: Identifier,
-  header: Header) {
-    clientType = provableStore.get(clientTypePath(id))
-    abortTransactionUnless(clientType !== null)
-    clientState = provableStore.get(clientStatePath(id))
-    abortTransactionUnless(clientState !== null)
-    clientType.checkValidityAndUpdateState(clientState, header)
+  clientMessage: ClientMessage) {
+    if err := clientState.VerifyClientMessage(clientMessage); err != nil {
+        return err
```

Comment:
`updateClient` has no return value. Shouldn't it call `abortTransactionUnless` instead?